docs/Monitoring Metrics with JMC - JDK Mission Control.md

```diff
@@ -0,0 +1,247 @@
+# Monitoring Metrics with JMC <small>(JDK Mission Control)</small>
+
+<!-- This Markdown file contains tabs to be rendered with https://jhildenbiddle.github.io/docsify-tabs/ -->
+
+<video muted autoplay loop style="width: 100%" poster="_media/frame.jpg">
+    <source type="video/mp4;codecs=vp9"
+        src="https://user-images.githubusercontent.com/83819/142078723-d1d52394-5f13-4b2d-a7b2-120bbc01013f.mp4">
+</video>
+
+# Introduction
+
+## Who is this for?
+
+This document is for contributors to Terasology—
+mainly developers and testers—
+who want a way to peek under the hood while the game is running.
+
+Some of these techniques can be used without a full Terasology development environment,
+but you should be comfortable running Java applications from the command line.
+
+## How is this different than Java Flight Recordings?
+
+If you've read [Analyzing JFR Recordings], you might be wondering how this is different.
+
+[Analyzing JFR Recordings]: https://github.com/Terasology/TutorialProfiling/wiki/Analyzing-JFR-Recordings
+
+Flight recordings are detailed logs of events over a short period of time.
+You can analyze a flight recording to learn about what happened during that time in the past.
+
+Here we'll be exploring a different feature of JMC, its JMX Console.<sup>1</sup>
+The JMX Console lets you see the _current state_ of a process.
+Instead of reading a log recorded by the application, it polls it for the current value of metrics you're interested in.
+
+<div style="display: flex; flex-wrap: wrap;">
+<div style="flex: 1 1 0;">
+
+**JFR**
+
+- 🖨 writes a log file
+- 📥 can be attached to a bug report for later review
+
+</div>
+<div style="flex: 1 1 0;">
+
+**JMX**
+
+- 🩺 connects to a live process
+- 📡 can connect to a remote server
+
+</div>
+</div>
+
+<!-- The line gets increasingly blurry as Java 14 added streaming features to JFC, and JFR-over-JMX is expected. -->
+
+#### Footnotes
+1. When you see “MBean Server” in Mission Control, it's referring to the same thing.
+Its JMX Console is the interface it provides to an application's MBean server. 
+
+
+# Prerequisite: Install JMC
+
+Download the latest stable release of [JDK Mission Control from AdoptOpenJDK](https://adoptopenjdk.net/jmc.html).
+
+Unpack the archive and run the program inside:`jmc`
+
+<!-- As of November 2021, Adoptium does not yet provide JMC. -->
+
+
+# Local Use
+
+## Connecting to Terasology
+
+The JVM Browser in JMC shows all Java processes running on your computer:
+
+<img src="_media/JMC/JMC JVM Browser.webp" width="599" height="271" alt="Each is listed with its main class and process identifier. Selecting a process reveals several options." >
+
+Choose the MBean Server for the Terasology process.
+<img src="_media/JMC/JMC JMX Console - Default.halfsize.webp" width="344" height="300" style="float: right;" alt="The dashboard shows current memory and CPU usage on speedometer-style gauges, and charts recent values on line graphs.">
```

Comment:
```suggestion
Choose the MBean Server for the Terasology process.

<img src="_media/JMC/JMC JMX Console - Default.halfsize.webp" width="344" height="300" style="float: right;" alt="The dashboard shows current memory and CPU usage on speedometer-style gauges, and charts recent values on line graphs.">
```

Add a line break here, otherwise it renders funny.

![image](https://user-images.githubusercontent.com/1448874/146416048-e2cee9fc-5822-4f92-9d57-148356a0130b.png)